packages/x-tree-view/src/internals/utils/extractPluginParamsFromProps.ts

```diff
@@ -28,6 +32,7 @@ export const extractPluginParamsFromProps = <
     rootRef,
     slots: slots ?? {},
     slotProps: slotProps ?? {},
+    apiRef: apiRef ?? {},
```

Comment:
I don't think we should initialize it
`{}` is not a valid value for `apiRef` which is the ref (of shape `{ current: XXX }`

Maybe I'm missing something, but I would just remove the fallback value here.